### DISCUSSION TITLE
Page buttons and field names not loading properly

### DISCUSSION THREAD

**Discussion by cloudam** [requarks/wiki#4195] [Help / Questions]
### Actual behavior
I installed wikijs using helm charts to my aks clusters. After installation while the page reloads to login page, all buttons inclusing the login details columns appearing weird.  like "filename.username", "filename.password", "action.create" etc Any idea what may be the cause. 

![image](https://user-images.githubusercontent.com/12157763/123461241-db016d00-d5b6-11eb-9b9d-22d1d39900fc.png)
![image](https://user-images.githubusercontent.com/12157763/123483500-cc29b300-d5d4-11eb-88d3-a89ce6a3786a.png)

Kubernetes version: 1.21.2
Helm version: helm 3.6.0
Wikijs: 2.5.201

**Comment by NGPixel** [comment]
Make sure that Wiki.js has internet access to download locale files. Otherwise sideload locale files as explained in the docs.
